refactor_me.py

```diff
@@ -3,15 +3,14 @@
 from collections import namedtuple, defaultdict
 from operator import itemgetter
 
-Expense = namedtuple('Expense', ('type_', 'amount'))
+Expense = namedtuple('Expense', 'expense_name amount')
 
 
 def sum_expenses(expenses, min_amount=0):
     aggregated_expenses = defaultdict(int)
     for expense in expenses:
-        amount = expense.amount
```

Comment:
Things to consider for this change:
1. What is faster access a data attribute twice or put it once to a variable and then access twice from the variable?
2. If there is a exception in the code some tools like Sentry dump local variables in the stacktrace. for this case namedtuple repr would work well and we will see `expense.amount` value, but in other cases putting to a variable first would work better
3. Some IDEs like PyCharm easily highlight standalone variable usages (when you put cursor on it). Will it work for data attributes too?